documentation/prerequisites.md

```diff
@@ -4,24 +4,40 @@ title: Prerequisites
 
 Following is the list of prerequisites for running Kubernetes with vSphere Cloud Provider:
 
-* Kubernetes version - 1.6.5 onwards. Tested with v1.6.5, v1.7, v1.8. 
-* vSphere version - 6.0.x onwards.
-* Container Host operating systems - 
-    - Photon - v1.0 GA 
-    - Ubuntu - v16.04 
-    - CoreOs - v4.11.2 
+* We recommend Kubernetes version 1.6.5 and above
+
+Following table summarizes major feature introduction in Kubernetes releases. This can help identify Kubernetes version for specific vSphere Cloud Provider feature.
+
+| Kubernetes Release | vSphere Cloud Provider feature |
```

Comment:
Kubernetes Release can be renamed to Kubernetes Version. Then we can drop v from all version numbers below.

Reply:
All deployment methods (kubeadm, kube-anywhere) needs v prefix in version. 
Thus let's be consistent and keep prefix v.

Reply:
Ok, sounds good.